spring-cloud-gcp-samples/spring-cloud-gcp-secretmanager-sample/src/main/resources/application.properties

```diff
@@ -0,0 +1,15 @@
+# You can directly load the secret into a variable like this example
+# This demonstrates multiple ways of loading the same application secret using template syntax.
+#
+# Please refer to the documentation for the protocol syntax.
+#
+# Examples:
+# my-app-secret-3=${sm://projects/<YOUR_PROJECT_ID>/secrets/application-secret}
+# my-app-secret-4=${sm://projects/<YOUR_PROJECT_ID>/secrets/application-secret/version/latest}
+# my-app-secret-5=${sm://<YOUR_PROJECT_ID>/application-secret/latest}
+
+# Defaults to latest version if omitted.
+my-app-secret-1=${sm://application-secret}
+
+# Use to specify explicit version (like version 1) under the default project.
+my-app-secret-2=${sm://application-secret/1}
```

Comment:
Similar to my comment above - is the `version/` necessary or not? It may be _clearer_, in this case, to include it.

Also noting here that the project ID is missing - does this pickup the `spring.cloud.gcp.project-id` if the project ID is omitted?

Reply:
Done -- see above; hopefully this clarifies things.

Yeah, if no project ID settings are set, then the project is inferred from Application Default Credentials. The user may override that default project id via: `spring.cloud.gcp.secretmanager.project-id` in `bootstrap.properties`.